manifests/pulp.pp

```diff
@@ -0,0 +1,61 @@
+# Katello configuration for pulp
+class katello::pulp {
+  include ::certs::qpid_client
+  # Because we re-use the CRL file
+  include ::katello::candlepin
```

Comment:
It does look that way. I haven't looked into having a different method for the CRL file and suggestions are highly welcome.